app/models/solidus_afterpay/payment_method.rb

```diff
@@ -6,9 +6,7 @@ class PaymentMethod < SolidusSupport.payment_method_parent_class
     preference :secret_key, :string
     preference :deferred, :boolean
     preference :popup_window, :boolean
-    preference :minimum_amount, :decimal
```

Comment:
here I think it's better to remove these preferences in a different commit. And remember to remove them also in the spec 😄

Reply:
Done!